system-int-test/test/cmr/system_int_test/search/collection_relevancy/collection_temporal_relevancy.clj

```diff
@@ -126,3 +126,46 @@
          (index/wait-until-indexed)
          (is (d/refs-match-order? [coll8 coll3 coll7 coll4 coll1 coll2 coll5 coll6]
                                   (search/find-refs :collection {:keyword "coll"}))))))))
+
+(deftest relevancy-multiple-temporal-ranges
+  (dev-sys-util/eval-in-dev-sys `(query-to-elastic/set-sort-use-temporal-relevancy! true))
```

Comment:
Is this not the default? It doesn't look like this config param was added with this PR. If it is not the default we should make sure to set it back to false at the end of the test.

Reply:
It is the default. Should I just remove it from both sets of tests?